api/v1beta1/kafkacluster_types.go

```diff
@@ -280,8 +280,12 @@ type EnvoyConfig struct {
 	NodeSelector map[string]string   `json:"nodeSelector,omitempty"`
 	Tolerations  []corev1.Toleration `json:"tolerations,omitempty"`
 	// Annotations defines the annotations placed on the envoy ingress controller deployment
-	Annotations              map[string]string `json:"annotations,omitempty"`
-	LoadBalancerSourceRanges []string          `json:"loadBalancerSourceRanges,omitempty"`
+	Annotations map[string]string `json:"annotations,omitempty"`
+	// If specified and supported by the platform, this will restrict traffic through the cloud-provider
+	// load-balancer will be restricted to the specified client IPs. This field will be ignored if the
```

Comment:
**Recommended**: we **SHOULD** improve phrasing for clarity. We **SHOULD** also regenerate the CRDs, samples, etc. if changed.

```suggestion
	// If specified and supported by the platform, traffic through the 
	// cloud-provider load-balancer will be restricted to the specified client
	// IPs. This field will be ignored if the
```